__tests__/app.test.js

```diff
@@ -155,4 +155,106 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("POST /api/articles/:article_id/comments", () => {
+        test("201: should return 200 and insert/post to add new record to the comments table", () => {
+            return request(app)
+                .post("/api/articles/9/comments")
+                .expect(201)
+                .send({
+                    user_name: "butter_bridge",
+                    body: {
+                        article_id: 9,
+                        author: "butter_bridge",
+                        votes: 20,
+                        created_at: new Date(),
+                        body: "post request will going to be ok",
+                    },
+                })
+                .then(({ body }) => {
+                    const { comments } = body;
+                    expect(comments).toHaveProperty("comment_id", expect.any(Number));
+                    expect(comments).toHaveProperty("body", expect.any(String));
+                    expect(comments).toHaveProperty("article_id", expect.any(Number));
```

Comment:
In this case we actually do know what the article id should be as we are posting a comment to a specific article, so this assertion can be made more precise.